node/src/actors/chain_manager/handlers.rs

```diff
@@ -834,6 +834,35 @@ impl PeersBeacons {
             .collect()
     }
 
+    /// Remove beacons that are old or has a different hash if is the same super epoch
+    pub fn remove_no_consensus_beacons(&mut self, superbeacon: CheckpointBeacon) {
```

Comment:
Remove is a misleading name because the beacons are simply set to "None". A better word would be "ignore".